### PR TITLE
Update Mapbox Assembly package

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Update from `@mapbox/mapbox-assembly` to `@mapbox/mbx-assembly`.

### CODE REVIEW

File: src/components/application-wrapper.js [hunk 1]
```diff
@@ -1,15 +1,8 @@
 import React from 'react';
 import { AppContext } from '../context.js';
 
-// Disable Raven if this isn't a production build, so we don't send development
-// errors to Sentry.
-if (process.env.DEPLOY_ENV !== 'production' && typeof window !== 'undefined') {
```
Comment (colleenmcginnis) [inline src/components/application-wrapper.js]: Is this is still applicable @davidtheclark?

**Comment by colleenmcginnis** [comment]
👋 @langsmith can you try running `npm install` and `npm start` on this branch? This should take care of the issues that were preventing you from running the site locally. Please let me know if you run into any trouble!

File: src/components/application-wrapper.js (hunk 1 above)
Comment (colleenmcginnis) [reply src/components/application-wrapper.js]: Perf. Thanks @davidtheclark! 🍕

**Comment by colleenmcginnis** [comment]
@langsmith this is ready for another try!

**Comment by langsmith** [comment]
Working for me on this branch! 🚀

**Pull request merged**